ansible_collections/arista/avd/molecule/ansible_only/inventory/group_vars/DC1_SERVERS.yml

```diff
@@ -0,0 +1,717 @@
+---
```

Comment:
should we trim this to only contain what we actually test on the few devices we have moved? It will never change if we don't fix it now.

Reply:
Same for services, it seems like there are no inline jinja being tested there.

Reply:
yeah let me investigate further

Reply:
cleaned up